_includes/master/reqs-sys.md

```diff
@@ -0,0 +1,32 @@
+## Node hosts
+
+- AMD64 processor
+
+- Linux kernel 3.10 or later. The following distributions are known to work 
+  well with {{site.prodname}} and {{include.orch}}.
+  - RedHat Linux 7 or later{% if include.orch == "Kubernetes" or include.orch == "host protection" %}
+  - CentOS 7 or later
+  - CoreOS Container Linux stable
+  - Ubuntu 16.04 or later
+  - Debian 8 or later
+  {% endif %}
+  {% if include.orch == "OpenShift" %}
+  - CentOS 7 or later
+  {% endif %}
+  {% if include.orch == "OpenStack" %}
+  - Ubuntu 16.04 or later
+  {% endif %}
+{% if include.orch == "Kubernetes" %}
+<br>
+- No other pod networking configurations stored in `/etc/cni/net.d`.
+{% endif %}
+
+## Key/value store
+
+{{site.prodname}} {{page.version}} requires an [etcdv3 or later instance](https://github.com/coreos/etcd/releases) accessible by all nodes in the cluster. 
+{% if include.orch == "Kubernetes" %} You can configure {{site.prodname}} to 
+access etcd directly or via the Kubernetes API.{% endif %}{% if include.orch == "OpenShift" %}
+Your nodes must access etcd directly, not via the Kubernetes API.{% endif %}
+{% if include.orch == "OpenStack" %}If you don't already have an etcd instance
```

Comment:
This was meant to match with the preceding sentence. I've changed it to "server" let me know if it reads any better. It's just a bit more specific than "key/value store" so that is why I prefer it. But let me know your thoughts on this.

Reply:
Server is OK.